code/modules/paperwork/folders.dm

```diff
@@ -33,8 +35,8 @@
 
 /obj/item/folder/examine()
 	. = ..()
-	if(length(contents))
-		. += span_notice("Right-click to remove [contents[1]].")
+	if(length(contents) && !contents_hidden)
+		. += span_notice("<b>Right-click</b> to remove [contents[1]].")
```

Comment:
would be super cool if these could also be screentips but i would prefer you leave that for a separate project